dotnet/src/Connectors/Connectors.Anthropic/Core/Models/Message/AnthropicImageContent.cs

```diff
@@ -4,31 +4,19 @@
 
 namespace Microsoft.SemanticKernel.Connectors.Anthropic.Core;
 
-/// <summary>
-/// Represents the request/response content of Claude.
-/// </summary>
-internal sealed class ClaudeMessageContent
+internal sealed class AnthropicImageContent : AnthropicMessageContent
```

Comment:
As this is not a KernelImageContent, suggest using something more related to another concept as `ImageMessage` as this is also a `Message`.

You sure the Image has all parameters of a message?

```suggestion
internal sealed class AnthropicImageMessageContent : AnthropicMessageContent
```